config.sample.py

```diff
@@ -20,5 +20,8 @@
 # CSH_LDAP credentials
 LDAP_BIND_DN = ''
 LDAP_BIND_PW = ''
+MAIL_USERNAME = ''
 MAIL_PASSWORD = ''
 MAIL_SERVER = ''
+MAIL_PORT = 465
+MAIL_USE_SSL = True
```

Comment:
Can we get consistent EOFs?

Reply:
EOF?

Reply:
End Of File. Either have a newline at the end of every file, or don't.

Right @Mstrodl ?

Reply:
Please have a newline at the end of every file. This is part of the unix definition of a file and expected by a number of tools.